agent/src/beerocks/fronthaul_manager/monitor/monitor_thread.cpp

```diff
@@ -10,7 +10,9 @@
 
 #include <bcl/beerocks_utils.h>
 #include <bcl/network/network_utils.h>
-#include <easylogging++.h>
+
+#define BEEROCKS_CUSTOM_LOGGER_ID BEEROCKS_MONITOR
+#include <bcl/beerocks_logging_custom.h>
```

Comment:
As noted elsewhere:
```suggestion

#define ELPP_DEFAULT_LOGGER BEEROCKS_MONITOR
#include <easylogging++.h>
```

Also, just to be sure that easylogging++.h is not included earlier in any of the already-included header files, move the definition before *all* includes.